client/editEvents.js

```diff
@@ -201,10 +201,10 @@ Template.editEvents.helpers({
     return startDate.fromNow() + " (" + rangeStr + ")";
   },
   roundOpen: function() {
-    return this.status == wca.roundStatuses.open;
+    return this.isOpen();
```

Comment:
Can we just get rid of the `roundOpen` and `canCloseRound` helpers entirely and call `isOpen` and `isClosed` directly from the templates?